### PR TITLE
about menu and departed class changes.

### PR DESCRIPTION
**Pull request by wbprice**

### CODE REVIEW

**Comment by wbprice** [comment]
sidebar: .timeframe background is now blue rather than red.  will expect to set red background color on <3-5 minute arrivals?

File: js/busfinder.js [hunk 1]
```diff
@@ -384,7 +384,11 @@ $(function(){
 			$('.arrow > img').attr('src', './img/arrow-down.png');
 				
 			if(!mapShowing) {
-				var mapHeight = window.innerHeight - 228; //map height is height of screen less the height of about bar .schedule
+				var scheduleHeight = $('.schedule').height(),
+					headerHeight = $('.navbar').height(),
+					stopHeight = $('.stop-name').height(),
+					headHeight = $('.head-label').height();
+				var mapHeight = window.innerHeight - (headerHeight + scheduleHeight + stopHeight + headHeight); //map height is height of screen less the height of about bar .schedule
```
Comment (bschoenfeld) [inline js/busfinder.js]: Fix this code. Missing 'var' on lines 388 - 390. Commas instead of semicolons on lines 387 - 389.

File: js/busfinder.js [hunk 2]
```diff
@@ -370,7 +370,7 @@ $(function(){
 			}
 			
 			if(minutesToArrival < 0) {
-		        this.$el.addClass('departed');
+		        this.$el.find('.timeframe').addClass('departed');
```
Comment (bschoenfeld) [inline js/busfinder.js]: Don't need to use 'find'. You can do this.$('.timeframe').add...

Cool, huh?!

File: js/jquery.jpanelmenu.js [hunk 3]
```diff
@@ -143,7 +143,7 @@
 
 				if ( $('#jPanelMenu-style-master').length == 0 )
 				{
-					$('body').append('<style id="jPanelMenu-style-master">body{width:100%}.jPanelMenu,body{overflow-x:hidden}#jPanelMenu-menu{display:block;position:fixed;top:0;'+jP.options.direction+':0;height:100%;z-index:-1;overflow-x:hidden;overflow-y:scroll;-webkit-overflow-scrolling:touch}.jPanelMenu-panel{position:static;'+jP.options.direction+':0;top:0;z-index:2;width:100%;min-height:100%;background:' + bgColor + '}</style>');
+					$('body').append('<style id="jPanelMenu-style-master">.jPanelMenu,body{overflow-x:visible}#jPanelMenu-menu{display:block;position:fixed;top:0;'+jP.options.direction+':0;height:100%;z-index:-1;overflow-x:visible;overflow-y:visible;-webkit-overflow-scrolling:touch}.jPanelMenu-panel{position:static;'+jP.options.direction+':0;top:0;z-index:2;width:100%;min-height:100%;background:' + bgColor + '}</style>');
```
Comment (bschoenfeld) [inline js/jquery.jpanelmenu.js]: What's going on here? Is this something that can't be done in our code? I hate to modify third party code unless there is no other option.

**Comment by bschoenfeld** [comment]
@wbprice yes. I like the idea of setting the arrival times to red when they are close, between 0 and 5 minutes away. Let's do it.

I don't like the new color for the background on the route number and minutes to arrival. I think it should be the same  as the stop name, like it used to be. And maybe the text should be white to provide a higher contrast?

File: js/jquery.jpanelmenu.js (hunk 3 above)
Comment (wbprice) [inline js/jquery.jpanelmenu.js]: This was not a fruitful direction.  I'm going to roll it back.

File: js/busfinder.js (hunk 1 above)
Comment (wbprice) [inline js/busfinder.js]: JS allows you to set multiple variables in one go, so this is valid code.  It runs on localhost just fine.  Are there any performance issues caused by doing it this way (blocking vs non-blocking)?

File: js/busfinder.js (hunk 2 above)
Comment (wbprice) [inline js/busfinder.js]: Cool beans.

File: js/busfinder.js (hunk 1 above)
Comment (bschoenfeld) [inline js/busfinder.js]: Sure enough. You learn something new everyday ;)

I still think it looks weird, but if it's valid then keep it in. @zirman suggested a while ago that we follow the Google style guide. I would like to get in the habit, but have yet to do so...

**Comment by wbprice** [comment]
One last thing, my edits to busfinder.js follow the google style guide now.  I'm going to try to rebase this, as it's getting a little messy.

**Pull request closed without merging**